components/tidb_query_executors/src/runner.rs

```diff
@@ -511,7 +528,8 @@ impl<SS: 'static> BatchExecutorsRunner<SS> {
         // record count less than batch size and is not drained
         while record_len < self.stream_row_limit && !is_drained {
             let mut current_chunk = Chunk::default();
-            let (drained, len) = self.internal_handle_request(
+            // for now `handle_streaming_request` don't take up significant cpu resources.
```

Comment:
You can leave it as a TODO. Streaming coprocessor on TiKV is just not enabled in TiDB now.